packages/performance/src/services/remote_config_service.test.ts

```diff
@@ -20,7 +20,7 @@ import { SettingsService } from './settings_service';
 import { CONFIG_EXPIRY_LOCAL_STORAGE_KEY } from '../constants';
 import { setupApi, Api } from './api_service';
 import * as iidService from './iid_service';
-import { getConfig, isDestFl } from './remote_config_service';
```

Comment:
Let's also remove the definition of `isDestFl`: https://github.com/firebase/firebase-js-sdk/blob/0131e1fcade96853997fc04ba1fee0e8d40ed445/packages/performance/src/services/remote_config_service.ts#L271-L296

Reply:
Good catch, cleaned up!